server/plugin.go

```diff
@@ -267,6 +268,8 @@ func (p *Plugin) handleNotification(body io.Reader, channel *TeamChannel) {
 		p.sendPostNotification(p.createSNSMessageNotificationAttachment(notification.Subject, messageNotification), channel)
 		return
 	}
+
+	p.sendPostNotification(p.createSNSUnknownTypeMessage(notification.Subject, notification.Message), channel)
```

Comment:
I'm thinking this behavior should be configurable in a plugin setting

Reply:
@mickmister I'm not sure if I described this option correctly in Settings. Check, please